restful.go

```diff
@@ -165,7 +168,7 @@ func (c *APIClient) DoQuery(ctx context.Context, query string, args []driver.Val
 	request := QueryRequest{
 		SQL: q,
 		Pagination: Pagination{
-			WaitTime: 90,
+			WaitTime: c.WaitTimeSecs,
```

Comment:
MaxRowsInBuffer      int64
	MaxRowsPerPage       int64
	need push to the pagination options similar to https://github.com/databendcloud/databend-jdbc/blob/main/databend-jdbc/src/main/java/com/databend/jdbc/DatabendConnection.java#L558